src/Dict.h

```diff
@@ -931,25 +931,23 @@ template <typename T> class Dictionary
 	friend zeek::DictIterator<T>;
 	friend zeek::RobustDictIterator<T>;
 
+	void SetLog2Buckets(int value)
+		{
+		log2_buckets = value;
+		bucket_count = 1 << log2_buckets;
+		bucket_capacity = (1 << log2_buckets) + log2_buckets;
+		}
+
 	/// Buckets of the table, not including overflow size.
 	int Buckets(bool expected = false) const
```

Comment:
I didn't see a `Buckets(true)` caller, so maybe the expected parameter can be removed?